packages/@okta/i18n/src/properties/login_cs.properties

```diff
@@ -59,116 +59,116 @@ errors.E0000047 = Byl prekrocen maxim\u00E1ln\u00ED pocet po\u017Eadavku. Zkuste
 errors.E0000079 = Operace nen\u00ED povolena. Chcete-li pokra\u010Dovat, obnovte str\u00E1nku.
 
 # ERROR MESSAGE TRANSLATIONS
-errors.E0000001 = Platnost Api selhala: {0}
+errors.E0000001 = Ov\u011B\u0159en\u00ED rozhran\u00ED API se nezda\u0159ilo: {0}
```

Comment:
FYI. All updates to existing Czech translations is related to quality improvement for legacy translation.